src/activitylog/activitylog.html

```diff
@@ -30,13 +30,19 @@ <h2>Activity Logs</h2>
         <div class="filter-wrapper">
           <filter-keyword></filter-keyword>
           <filter-option filter-key="id">
-            Select extensions to filter logs
+            Select extensions to filter
           </filter-option>
           <filter-option filter-key="viewType">
-            Select view type to filter logs
+            Select view type to filter
           </filter-option>
           <filter-option filter-key="type">
-            Select api type to filter logs
+            Select api type to filter
+          </filter-option>
+          <filter-option filter-key="name">
+            Select api name to filter
+          </filter-option>
+          <filter-option filter-key="other">
```

Comment:
This is not what I meant.

> Did you mean to make another filter option named other where there will be content script urls (for now) as checkboxes?

We've misunderstood each other. I thought that you suggested to add an option with value "other" in the existing `<filter-option filter-key="name">`.

I meant this:

```
<filter-option filter-key="name">
[x] tabs.create
[x] tabs.update
[x] other <-- New option here that matches content script logs.
</filter-option>
```


... where `other` covers everything else that doesn't have an API name. Or, since the only type of logs are content scripts, call it something like "Content scripts".

Reply:
@Rob--W we already get a checkbox named `content_script` inside API Type `filter-option` when any content script is found. Do we want a checkbox with named  "Content scripts" under API Name `filter-option`  too ?

Reply:
Currently your filter design assumes that the set of options is closed. That is, every log entry matches one of the given options, and every option matches at least one log.

If the UI is clear* that "API name" only matches log entries related to API events (this is indeed reasonable), then you could indeed choose to not add an "other" or "content scripts" checkbox at all.

\* one stupidly simple way to achieve this is by adding a tooltip that explains to which log types the "API name" filter applies.